src/ontology/mondo-edit.obo

```diff
@@ -151243,6 +151241,11 @@ xref: UMLS:CN204412 {source="MONDO:equivalentTo", source="MEDGEN:kboom-pr94-c94"
 xref: UMLS:CN536252 {source="MONDO:equivalentTo", source="MEDGEN:kboom-pr94-c94"}
 is_a: MONDO:0002473 {source="MESH:C536137/inferred", source="NCIT:C123171", source="indirect"} ! cystic kidney disease
 is_a: MONDO:0019741 {source="Orphanet:34149"} ! familial cystic renal disease
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/11630 {source="Orphanet:34149", source="PMID:25738250"} ! HNF1B
```

Comment:
remember owl semantics. You are saying every instance of this disease is caused by thes4 genes

Reply:
I removed this, per our conversation, thanks!